Formula/blast.rb

```diff
@@ -34,11 +44,26 @@ def install
   end
 
   test do
+    output = shell_output("#{bin}/update_blastdb.pl --showall")
+    assert_match "nt", output
+
+    # output = shell_output("#{bin}/get_species_taxids.sh -t 9606")
+    # assert_match "9606", output
+
     (testpath/"test.fasta").write <<~EOS
       >U00096.2:1-70
       AGCTTTTCATTCTGACTGCAACGGGCAATATGTCTCTGTGTGGATTAAAAAAAGAGTGTCTGATAGCAGC
     EOS
     output = shell_output("#{bin}/blastn -query test.fasta -subject test.fasta")
     assert_match "Identities = 70/70", output
+
+    # Create BLAST database
+    output = shell_output("#{bin}/makeblastdb -in test.fasta -out foo -title FOO -dbtype prot")
+    assert_match "Adding sequences from FASTA", output
+
+    # Check newly created BLAST database
+    output = shell_output("#{bin}/blastdbcmd -info -db foo")
+    assert_match "Database: FOO", output
```

Comment:
```suggestion
    assert_match "Database: test.fasta", shell_output("#{bin}/blastdbcmd -info -db test")
```
When I run this command locally, I get the error:
```console
$ blastdbcmd -info -db test
Error: mdb_dbi_open: MDB_NOTFOUND: No matching key/data pair found
```
Does it work for you?

Reply:
I don't see any errors.